app/src/main/java/com/gholem/moneylab/features/editTransaction/viewmodel/EditTransactionViewModel.kt

```diff
@@ -86,8 +86,10 @@ class EditTransactionViewModel @Inject constructor(
         Action.GetCurrentCategory(categories[result.toInt()]).send()
     }
 
-    fun onDoneButtonClick() {
-        Action.SetEditedTransaction.send()
+    fun onDoneButtonClick(amount: Editable?) {
```

Comment:
Why it's still `Editable`? As I said, don't pass `Editable` to the `ViewModel`, change it to `String`. Also change it to not nullable value and use `isBlank()` method instead of `isEmpty()` (in that case it will be `isNotBlank()`)